packages/@react-aria/focus/test/FocusScope.test.js

```diff
@@ -779,6 +779,42 @@ describe('FocusScope', function () {
       expect(document.activeElement).toBe(item2);
     });
 
+    it('should move focus forward and allow users to skip certain elements', function () {
+      function Test() {
+        return (
+          <FocusScope>
+            <Item data-testid="item1" />
+            <Item data-testid="item2" data-skip />
+            <Item data-testid="item3" />
+          </FocusScope>
+        );
+      }
+
+      function Item(props) {
+        let focusManager = useFocusManager();
+        let onClick = () => {
```

Comment:
We are testing the skipping via a function call based on an attribute, correct? We don't need to have a keyboard version of these tests, right?

Reply:
we have other tests that check keyboard behavior, the line of code we are testing is executed in both flows in the same way, so I didn't see it as a necessary to include more